LiteDB/Engine/Disk/StreamFactory/FileStreamFactory.cs

```diff
@@ -17,13 +17,15 @@ internal class FileStreamFactory : IStreamFactory
         private readonly string _password;
         private readonly bool _readonly;
         private readonly bool _hidden;
+		private readonly bool _useAesStream;
 
-        public FileStreamFactory(string filename, string password, bool readOnly, bool hidden)
+		public FileStreamFactory(string filename, string password, bool readOnly, bool hidden, bool useAesStream = true)
```

Comment:
This is a breaking change, instead of adding one more parameter on the ctor, please create a new one with this parameter, and use `this` if possible to forward to the old one

Reply:
The class is internal. No breaking.

Reply:
Ops, my bad